flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -303,6 +303,16 @@
 	// ------------------------ File System Behavior ------------------------
 
 	/**
+	 * Key to specify the default filesystem to be used by a job, e.g.
+	 * <code>file:///</code> which is the default (see {@link ConfigConstants#DEFAULT_FILESYSTEM_SCHEME})
```

Comment:
Same questions as wth the docs. Why `file:///`? And I would remove the `HDFS deployment` part at the end.

Furthermore, I would add something alone the lines of `This scheme is used <b>ONLY</b> if no other scheme is specified in the user-provided <code>URI</code>` to the docs as well.